### PR TITLE
Adds CG Education Part 1

### PR DESCRIPTION
**Pull request by castavridis**

### CODE REVIEW

**Comment by castavridis** [comment]
@ktmathews89 , please update project files link to `.zip` path after this branch is merged in.

File: lesson_9/cg_ed_1.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+2. Use Google to solve problems!
+3. Become more comfortable using Developer Tools, specifically Console.
+4. *Frequently* use JavaScript's built-in `console` object to test and debug your work.
+5. Create some variables.
+6. Do basic math with those variables.
+7. Learn how to use resources like MDN to discover more functions you can use for built-in JavaScript objects and data types. 
+
+## Getting Started
+
+First, download the project files [here](https://bit.ly/cg_ss18_cg-education). The HTML and CSS files have been written for you so you can focus on the JavaScript. However, you are free to update the HTML and CSS as you like! Just remember that the focus of this project is the JavaScript. You'll find basic wireframes for the site design in the `wireframes/` directory.
```
Comment (castavridis) [inline lesson_9/cg_ed_1.md:33]: Here's the project files link!

File: lesson_9/cg_ed_1.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+permalink: /cg_ed_1
+---
+
+# CG Education - JavaScript Part I
+
+**Plan:** List steps/tasks in [Trello Boards](https://trello.com/cg_webdev_ss_2018)
+
+**Do:**  Download app to your phone and play (instructions below).
+
+**Record:** [Lesson 9.3 CoderGirl Education Project Part 1](https://learn.launchcode.org/courses/131/assignments/7223)
```
Comment (castavridis) [inline lesson_9/cg_ed_1.md]: @ktmathews89 please update with the real link, I forgot!

Comment (tigeranniemac) [reply lesson_9/cg_ed_1.md]: Created a new lesson on Canvas for lesson 9

**Pull request merged**